CONTRIBUTING.md

```diff
@@ -78,14 +86,6 @@ cd optimism
 yarn install
 ```
 
-### Install the Correct Version of NodeJS
```

Comment:
Oh I see this used to be down here.    Yea I don't think this changes works because you can't run nvm use until after you clone.   You can consider moving `yarn install` down

Reply:
Good point! Just moved it down in the latest commit.